main.go

```diff
@@ -89,42 +99,104 @@ func main() {
 				},
 			},
 			{
-				Name:    "add git",
-				Aliases: []string{"git", "addgit"},
-				Description: `Allows canaveral to use your git credentials for repo management.
+				Name:    "add github credentials",
+				Aliases: []string{"gh", "github", "addgh", "addgithub"},
```

Comment:
For this, did you mean to use the alias `agithub`? I'm just guessing cuz below you used `rgithub`, and the alias `github` is a bit generic/vague